spec/functional/resource/insserv_spec.rb

```diff
@@ -1,206 +0,0 @@
-# encoding: UTF-8
-#
-# Author:: Dheeraj Dubey (<dheeraj.dubey@msystechnologies.com>)
-# Copyright:: Copyright 2009-2019, Chef Software, Inc.
-# License:: Apache License, Version 2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-require "spec_helper"
-require "functional/resource/base"
-require "chef/mixin/shell_out"
-require "fileutils"
-
-describe Chef::Resource::Service, :requires_root, :sles11 do
```

Comment:
Can you see if this can potentially run on Ubuntu?

Reply:
Sure, @tas50 will test that.

Reply:
Hi @tas50 We've checked tests are not running in Ubuntu16 & Ubuntu18 but with little change in `initest` script that are supporting for the tests they run on Opensuse15.